lib/content-services/breadcrumb/breadcrumb.component.ts

```diff
@@ -179,7 +180,7 @@ export class BreadcrumbComponent implements OnInit, OnChanges, OnDestroy {
 
     onRoutePathClick(route: PathElementEntity, event?: Event): void {
         if (event) {
-            event.preventDefault();
+            // event.preventDefault();
```

Comment:
commented out code

Reply:
ups, I forgot about that one, I will fix it